src/QMCHamiltonians/SOECPotential.h

```diff
@@ -79,6 +79,10 @@ class SOECPotential : public OperatorBase
   //return a shared resource to a collection
   void releaseResource(ResourceCollection& collection, const RefVectorWithLeader<OperatorBase>& o_list) const override;
 
+  //triggers using fast evaluation using TWFFastDerivWrapper for normal evalualations and derivatives.
+  void setFastEvaluation(bool fast) { use_fast_evaluation_ = fast; }
+  TWFFastDerivWrapper psi_wrapper_;
```

Comment:
psi_wrapper_ is no longer used right? Please remove.

Reply:
fixed